xsd/pmo/types.xsd

```diff
@@ -46,4 +46,9 @@ SOFTWARE.
       <xs:minInclusive value="0"/>
     </xs:restriction>
   </xs:simpleType>
+  <xs:simpleType name="repo">
+    <xs:restriction base="xs:string">
+      <xs:pattern value="[A-Z0-9a-z.\-/]{3,}"/>
```

Comment:
@carlosmiranda github repo usually is a combination of github username + / + repo name.
Regexp for github username is `/^[a-z\d](?:[a-z\d]|-(?=[a-z\d])){0,38}$/i` (javascript syntax, from https://github.com/shinnn/github-username-regex#githubusernameregex)

I'm not sure what is the repo name, we could assume it is similar to username for now.